Update staging copy documentation (#4900)
mozilla/foundation.mozilla.org: docs/local_development.md

> Some development work requires testing changes against "whatever the current production database looks like", which requires having postgresql installed locally (`brew install postgresql` on mac; download and run the official installer for windows; if you use linux/unix, you know how to install things for your favourite flavour, so just do that for postgresql). We backport prod data to staging every week, scrubbing PII, so we'll be creating a copy of that for local testing, too.

Reviewer: We probably want to add a bit to make it explicit this relies on the Heroku cli, with a link to its download page.

Author: yeah, I'll add in something to that effect.

> **Note**: your postgres version must be compatible with the version that is used on heroku in order for the `pg_dump` command to work. In general, this means that the result of `psql --version` must be **greater or equal to** `heroku pg:info -a foundation-mofostaging-net` (look for "PG Version")

Reviewer: ```suggestion
**Note**: your postgres version must be compatible with the version that is used on heroku in order for the `pg_dump` command to work. In general, this means that the result of `psql --version` must be **greater or equal to** the version found when running `heroku pg:info -a foundation-mofostaging-net` (look for "PG Version")
```

The steps involved in cloning the database for local use are as follows:

> 1. Start the `postgres` service named in the docker-compose file with `docker-compose up postgres`. If you want to detach stdout, add the `-d` flag to the command.

Reviewer: ```suggestion
1. Run `docker-compose up postgres` to start the `postgres` service without starting the rest of the server setup (note that if you want to detach stdout, add the `-d` flag to the command)
```

> 2. Drop the existing `wagtail` database in the PostgreSQL server inside your docker environment (if it exists) with `dropdb -h localhost -p 5678 -U foundation wagtail`

Reviewer: ```suggestion
2. Drop the existing `wagtail` database in the PostgreSQL server inside your docker environment with `dropdb -h localhost -p 5678 -U foundation wagtail --if-exists`
```

Reviewer: But it might make more sense to tell people to use `pg:reset`? That way they won't even need to have postgress installed locally...
